selenium/base/Dockerfile

```diff
@@ -5,17 +5,18 @@ COPY xseld /xseld
 COPY fileserver /fileserver
 
 RUN \
+    if [ `uname -m` = "aarch64" ]; then ARCH="arm64"; else ARCH="amd64"; fi && \
     apt-get update && \
     apt-get install -y upx-ucl libx11-dev && \
     cd /xseld && \
-    GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" && \
+    GOOS=linux GOARCH=$ARCH go build -ldflags="-s -w" && \
     upx /xseld/xseld && \
     cd /fileserver && \
     go test -race && \
-    GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" && \
+    GOOS=linux GOARCH=$ARCH go build -ldflags="-s -w" && \
     upx /fileserver/fileserver
 
-FROM ubuntu:20.04
+FROM ubuntu:18.04
```

Comment:
Ubuntu 20.04 is available under https://hub.docker.com/r/arm64v8/ubuntu/tags. So it does not make sense to downgrade.

Reply:
@vania-pooh 
The reason why it was downgraded is that Ubuntu 20.04+ doesn't ship the latest Chromium updates: https://launchpad.net/ubuntu/+source/chromium-browser
By default, it comes with 80 version of Chromium, and `apt update` upgrades to 85 max. Moreover, I don't believe this situation will change in the future.
But there are recent Chromium updates on 18.04.
So I see 2 options here:
- leave 18.04 and monitor is latest Ubuntu versions would keep Chromium up-to-date before EOL of 18.04;
- create a workaround that takes binaries from 18.04 (not sure about complexity, compatibility or any other issues which may occur with this approach though).

Thoughts?

Reply:
I would add `ARG VERSION=20.04` to the first line of base image Dockerfile and use 18.04 for M1 images.